homeassistant/components/evohome.py

```diff
@@ -8,40 +8,37 @@
 """
 
 # Glossary:
-# TCS - temperature control system (a.k.a. Controller, Parent), which can
-# have up to 13 Children:
-#   0-12 Heating zones (a.k.a. Zone), and
-#   0-1 DHW controller, (a.k.a. Boiler)
+#   TCS - temperature control system (a.k.a. Controller, Parent), which can
+#   have up to 13 Children:
+#     0-12 Heating zones (a.k.a. Zone), and
+#     0-1 DHW controller, (a.k.a. Boiler)
+# The TCS & Zones are implemented as Climate devices, Boiler as a WaterHeater
 
 import logging
 
 from requests.exceptions import HTTPError
 import voluptuous as vol
 
+from homeassistant.components.climate.evohome import EvoController, EvoZone
```

Comment:
We shouldn't import anything from the platform to the component. That's the wrong way. Imports should go the other way, from component to platform.